requests_toolbelt/utils/dump.py

```diff
@@ -0,0 +1,166 @@
+"""This module provides functions for dumping information about responses."""
+import collections
+
+from requests import compat
+
+
+__all__ = ('dump_response',)
+_PrefixSettings = collections.namedtuple('PrefixSettings',
+                                         ['request', 'response'])
```

Comment:
I know this is weird, but I'd rather have this declaration closer to its subclass. I just don't love having them separated by an unrelated block of code.